components/contribution-flow/index.js

```diff
@@ -117,6 +118,7 @@ class ContributionFlow extends React.Component {
     this.state = {
       error: null,
       stripe: null,
+      elements: null,
```

Comment:
In this scope, calling this variable `elements` is not very explicit. Can we find a better name?

Reply:
Done. 👍🏽